main.py

```diff
@@ -20,7 +20,7 @@ def main():
     step_motion = StepperMotion(GPIO)
 
     # TODO: Example call
-    step_motion.mov(1, fwd_dist)
+    step_motion.mov(1, fwd_dist, GPIO)
```

Comment:
Constants like Forward(FWD) or Reverse(REV) should have a defined constant Boolean value in a separate file. Say FWD is True and Reverse is False. It makes it easier for other people to work with the code.